### PR TITLE
git-paste: add page

### PR DESCRIPTION
**Pull request by sebastiaanspeck**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/git-paste.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# git paste
+
+> Send commits to a pastebin site using pastebinit.
```
Comment (gutjuri) [inline pages/common/git-paste.md:3]:
```suggestion
> Send commits to a pastebin site using `pastebinit`.
```

File: pages/common/git-paste.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-paste>.
+
+- Send commits to a pastebin site using pastebinit:
```
Comment (gutjuri) [inline pages/common/git-paste.md:7]:
```suggestion
- Send commits to a pastebin site using `pastebinit`:
```

File: pages/common/git-paste.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+`git paste {{paste_url}}`
+
+- You can also pass options understood by git-rev-parse(1) in order to select a different set of commits:
```
Comment (gutjuri) [inline pages/common/git-paste.md:11]:
```suggestion
- Pass options understood by git-rev-parse(1) in order to select a different set of commits:
```

File: pages/common/git-paste.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-paste>.
+
+- Send commits to a pastebin site using `pastebinit`:
+
+`git paste {{paste_url}}`
+
+- Pass options understood by `git-rev-parse(1)` in order to select a different set of commits:
+
+`git paste @^ {{paste_url}}`
```
Comment (acuteenvy) [inline pages/common/git-paste.md:13]:
```suggestion
- Pass options to `git format-patch` in order to select a different set of commits (`@^` selects the parent of HEAD, and so the currently checked out commit is sent):

`git paste {{@^}}`
```

File: pages/common/git-paste.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# git paste
+
+> Send commits to a pastebin site using `pastebinit`.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-paste>.
+
+- Send commits to a pastebin site using `pastebinit`:
+
+`git paste {{paste_url}}`
```
Comment (acuteenvy) [inline pages/common/git-paste.md:9]:
```suggestion
- Send the patches between the current branch and its upstream to a pastebin using `pastebinit`:

`git paste`
```

**Comment by sebastiaanspeck** [comment]
> Looking at the man page, `git paste` doesn't seem to support specifying the `paste_url`.
> 
> 
> 
> > SYNOPSIS
> 
> > git-paste \<git format-patch options\>
> 
> 
> 
> > CONFIGURATION
> 
> > The pastebinit software chooses the most appropriate pastebin site for your operating
> 
> > system and the pastebinit documentation https://manpages.debian.org/pastebinit
> 
> > contains details of how to configure it to use a different site.

You are completely right. Mixed up the command with the output: https://github.com/tj/git-extras/blob/master/Commands.md#git-paste

**Review by acuteenvy — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**